### PR TITLE
F#2437 additional metadata source info

### PR DESCRIPTION
**Pull request by thinkcsww**
### Description

Add  addtional metadata information after explore
**Related Issue** : 

#2437 

### How Has This Been Tested?

1. Go to Explore Data Page using lnb
2. Click Data Card
3. Check Modal shows proper data
4. Click See all list Button
5. Check Modal shows proper data

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by ufoscw** [comment]
@thinkcsww 
please resolve conflicts.

**Comment by ufoscw** [comment]
run build
deploy to 3

File: discovery-frontend/src/app/meta-data-management/metadata/service/metadata.model.service.ts [hunk 1]
```diff
@@ -73,6 +73,41 @@ export class MetadataModelService extends AbstractService {
    * @param {Metadata} metadata
    */
   public setMetadata(metadata: Metadata): void {
+    // metadata.source.sourceInfo = {
```
Comment (ufoscw) [inline discovery-frontend/src/app/meta-data-management/metadata/service/metadata.model.service.ts]: @thinkcsww 
Please delete dummy code.

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!-- top -->
+<div class="ddp-top-flow">
+  <!-- 수정 data -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Please do not use a comment in Korean.

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  <div class="ddp-data-form">
+    <span class="ddp-txt-date">
+      {{ this.translateService.instant('msg.metadata.ui.recent.update', { modifiedTime: metadata.modifiedTime, modifiedBy: metadata.modifiedBy.fullName }) }}
+    </span>
+
+    <!-- more -->
+    <!-- more 버튼 클릭시 ddp-selected 추가 -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here,

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <a href="#" class="ddp-list-link"><em class="ddp-icon-delete"></em>{{ 'msg.metadata.metadata.detail.ui.delete.metadata.and.source' | translate }}</a>
+        <span class="ddp-data-by">
+          {{ this.translateService.instant('msg.metadata.ui.recent.update', { modifiedTime: metadata.modifiedTime, modifiedBy: metadata.modifiedBy.fullName }) }}
+        </span>
+      </div>
+      <!-- //popup -->
+    </div>
+    <!-- //more -->
+  </div>
+  <!-- //수정 data -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  <!-- navi wrap -->
+  <div class="ddp-wrap-navi">
+    <a href="javascript:" class="ddp-btn-back" (click)="goBack()"></a>
+    <!-- navi -->
+    <div class="ddp-ui-naviarea">
+      <!-- name -->
+      <!-- 클릭시 ddp-selected 추가 -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      <div class="ddp-box-navi" [class.ddp-selected]="editNameFlag" (click)="onClickEditName()">
+        <div class="ddp-wrap-naviname">
+              <span class="ddp-data-naviname">{{ editName }}
+                <em class="ddp-icon-edit2" ></em>
+              </span>
+        </div>
+        <!-- 편집 input -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="ddp-wrap-input">
+          <input type="text" class="ddp-input-navi" placeholder="" [(ngModel)]="editName" value="{{ editName }}">
+          <span class="ddp-btn-check" (click)="onClickUpdateEditingName()"></span>
+        </div>
+        <!-- //편집 input -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+      </div>
+      <!-- //name -->
+
+      <!-- description -->
+      <!-- 클릭시 ddp-selected 추가 -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+      <div class="ddp-box-navidet" [class.ddp-selected]="editDescriptionFlag" (click)="onClickEditDescription()">
+        <div class="ddp-wrap-navidet">
+              <span class="ddp-data-navidet">{{ metadata.description }}
+                <em class="ddp-icon-edit2"></em>
+              </span>
+        </div>
+        <!-- 편집 input -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here

File: discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="ddp-wrap-input">
+          <textarea type="text" class="ddp-input-navi" placeholder="" [(ngModel)]="editDescription" >{{ editDescription }}</textarea>
+          <span class="ddp-btn-check" (click)="onClickUpdateEditingDescription()"></span>
+        </div>
+        <!-- //편집 input -->
```
Comment (alchan-lee) [inline discovery-frontend/src/app/meta-data-management/metadata/component/metadata-detail-top.component.html]: Here too.

**Comment by thinkcsww** [comment]
@alchan-lee @ufoscw 
수정했습니다.

**Comment by ufoscw** [comment]
run build
deploy to 3

**Comment by ufoscw** [comment]
@thinkcsww 
1. when I change a description of metadata, patch api request error
DevTools_-_52_231_184_135_8187_app_v2_index_html

2. Tags and Catalogs Input not work.
metatron_Discovery

3. please remove <a> tag
metatron_Discovery

4. please remove "Previous" button
metatron_Discovery

**Comment by eltriny** [comment]
@ufoscw 
수정했습니다. 확인 부탁드립니다.

**Comment by ufoscw** [comment]
run build
deploy to 3

**Comment by ufoscw** [comment]
@thinkcsww 
1. Message is strange after modifying metadata description. (Datasource -> Metadata)
metatron_Discovery

2. description No line breaks
metatron_Discovery
metatron_Discovery

3. If no catalog is specified, it should be displayed as blank, not "Unclassified".
Text input instead of the add catalog button
metatron_Discovery

**Comment by eltriny** [comment]
@ufoscw 수정했습니다.

**Comment by ufoscw** [comment]
run build
deploy to 3

**Comment by ufoscw** [comment]
@thinkcsww 
1. There is no delete button.
2. typo in text input placeholder ("Search column" - > " Search catalog")
metatron_Discovery

**Comment by eltriny** [comment]
@ufoscw 수정했습니다.

**Comment by ufoscw** [comment]
run build
deploy to 3

**Comment by ufoscw** [comment]
run build
deploy to 3

**Pull request merged**